### ISSUE TITLE
yangster-app-electron doesn't work on Ubuntu and Windows

### ISSUE THREAD

**Issue by huyuwen** [theia-ide/yangster#68]
Hi,

I tried to only use yangster-app-electron/package.json to build desktop application of yangster.
It works fine to build it on macOS and runing successfully.

But when I tried the same steps on Ubuntu and Windows, none of the two platforms work.
It is fine to build yangster-app-electron/package.json on all the platforms with similar build logs.
```
yangster-app-electron\> ls
package.json
yangster-app-electron\> yarn
yarn install v1.7.0
...
Done in 352.32s.
```

But on Ubuntu, I got the following error message,
```
yangster-app-electron\> yarn theia start --no-cluster
yarn run v1.10.1
root INFO Theia app listening on http://localhost:35341.
...
...yangster-app-electron/node_modules/electron/dist/electron --no-cluster 
...yangster-app-electron/src-gen/frontend/electron-main.js: symbol lookup error: 
/.../yangster-app-electron/node_modules/vscode-nsfw/build/Release/nsfw.node: undefined symbol: _ZN4node19GetCurrentEventLoopEPN2v87IsolateE
Done in 12.81s.
```

And on Windows, I got similar error message.
```
 yangster-app-electron\> yarn theia start --no-cluster
Error: Invalid access to memory location.
\\?\C:\...yangster-app-electron\node_modules\vscode-nsfw\build\Release\nsfw.node
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:172:20)
    at Object.Module._extensions..node (module.js:598:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:172:20)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\...\yangster-app-electron\node_modules\vscode-nsfw\lib\src\index.js:3:16)
    at Object.<anonymous> (C:\...\yangster-app-electron\node_modules\vscode-nsfw\lib\src\index.js:130:3)
Done in 9.57s.
```
I tried with nodejs 8.9.4 and 8.12.0, yarn 1.7.0 and 1.10.1. None of them work on Ubuntu or Windows.
I'm not sure whether this is a bug or do I miss any dependencies or configurations?

**Comment by kittaakos** [comment]
After updating to the `next` Theia, I am hitting [this error](https://github.com/Axosoft/nsfw/issues/25) on Windows.

**Comment by huyuwen** [comment]
Alright,, finally, I would like to have a bunled electron application, but not locally from source.
I will try to build and run yangster-electron project. I will double check and comparing the differences first.
Regarding to running application from source, I think it works on OS X, I tried that. 
And I just tried to build and run from local source on Ubuntu, it works fine.

Well, then we have to wait for the bug fix for the mentioned error on Windows.
